ch390/examples/iperf/main/idf_component.yml

```diff
@@ -0,0 +1,8 @@
+description: iperf Benchmark Example
+dependencies:
+  ch390:
+    override_path: '../../../'
+  cmd_system:
+    path: ${IDF_PATH}/examples/system/console/advanced/components/cmd_system
+  iperf:
+    path: ${IDF_PATH}/examples/common_components/iperf
```

Comment:
iperf is now migrating to separate repo. Please use [iperf component for Component Registry](https://components-staging.espressif.com/components/espressif/iperf?language=en).